### ISSUE TITLE
~ reserved word

### ISSUE THREAD

**Issue by kamaydeo** [jirutka/rsql-parser#20]
This is more of a question. I noticed that ~ is reserved but I don't see it in the supported operators. I would like to use ~ for fuzzy searches e.g q=name=~bob

However, this is allowed since it's a reserved keyword. Can you explain the use for "~"?

**Comment by kaush** [comment]
I agree. I wanted to create LIKE(~~), NOT_LIKE(!~~) etc, but ComparisonOperator won't allow it. 
Tilde (~) fits naturally with regex type searches and is URL safe.

**Comment by jirutka** [comment]
> Tilde (~) fits naturally with regex type searches and is URL safe.

That’s the exact reason why it’s reserved, so I can add `~=` operator in future.

**Comment by mstruensee** [comment]
any update for adding ~= ?
